server/src/main/java/org/elasticsearch/search/aggregations/bucket/adjacency/AdjacencyMatrixAggregatorFactory.java

```diff
@@ -50,9 +50,9 @@ public AdjacencyMatrixAggregatorFactory(String name, List<KeyedFilter> filters,
         keys = new String[filters.size()];
         for (int i = 0; i < filters.size(); ++i) {
             KeyedFilter keyedFilter = filters.get(i);
-            this.keys[i] = keyedFilter.key();
+            keys[i] = keyedFilter.key();
```

Comment:
These were confusing because above they are referred to without `this` but below they had it.